tests/pytests/unit/grains/test_core.py

```diff
@@ -1163,7 +1138,7 @@ def test_windows_platform_data():
         "productname",
         "serialnumber",
         "timezone",
-        "virtual",
+        # "virtual", <-- only present on VMs
```

Comment:
Why are we removing virtual if it was working previously?

Reply:
It's not needed for testing. The grain is only available on virtual machines. We don't need to test it for everything.